test/factories/questions.rb

```diff
@@ -1,7 +1,7 @@
 FactoryBot.define do
   factory :question do
     title { FFaker::HipsterIpsum.sentence }
-    is_private { false }
+    is_private { [true, false].sample }
```

Comment:
I feel all things should be randomized unless they are required, it makes sure we don't couple to them without realizing it. In other words it forces us to explicitly pass in each thing that matters.